WordPress/lint.xml

```diff
@@ -83,10 +83,6 @@
         <ignore regexp="java-common-1.13.jar" />
     </issue>
 
-    <issue id="IconLauncherFormat">
-        <ignore path="src/main/res/mipmap-anydpi-v26/app_icon.xml" />
-    </issue>
```

Comment:
Its worth mentioning that I removed this because I got this error after upgrading:
```
Errors found:

/Users/james/src/tmp/WordPress-Android/WordPress: Error: Unknown issue id "IconLauncherFormat", found in /Users/james/src/tmp/WordPress-Android/WordPress/lint.xml [LintError]
```

So it is not necessary to ignore this issue anymore.